rust/connlib/tunnel/src/peer.rs

```diff
@@ -233,6 +233,17 @@ impl ClientOnGateway {
         self.recalculate_filters();
     }
 
+    pub(crate) fn retain_authorizations(&mut self, authorization: BTreeSet<ResourceId>) {
+        for (resource, _) in self
+            .resources
+            .extract_if(|resource, _| !authorization.contains(resource))
+        {
+            tracing::info!(%resource, "Revoking resource authorization");
```

Comment:
```suggestion
            tracing::debug!(%resource, "Revoking resource authorization");
```

This could be quite noisy (thousands of messages) if the Gateway hasn't connected in a while, or connects after a major access change such as an IdP deletion.

Reply:
I think it is still a major state change of the system so INFO seems to be justified.